pkg/network/tags_linux.go

```diff
@@ -9,12 +9,12 @@
 package network
 
 import (
-	netebpf "github.com/DataDog/datadog-agent/pkg/network/ebpf"
+	"github.com/DataDog/datadog-agent/pkg/network/http"
 )
 
 // GetStaticTags return the string list of static tags from network.ConnectionStats.Tags
 func GetStaticTags(staticTags uint64) (tags []string) {
-	for tag, str := range netebpf.StaticTags {
+	for tag, str := range http.StaticTags {
```

Comment:
What happens when you have tags that are not HTTP-specific, or even USM-specific?

Reply:
ATM, there are no tags besides those
so if we will have more we should add it here
WDYT?

Reply:
Maybe every package can have its own tags? We would have to go over each collection separately here. Downside is that we don't have them in one place so there could be (unintended) duplication.

Reply:
They will add it to their package and they will add here the iteration on the tags
Or what hasan proposed seems good as well

Reply:
Do you have a suggestion to make it better?

Reply:
I think the original intent of having it `pkg/network/ebpf` was to have a common place, but your idea(s) seem fine. We can always iterate on it, in the future.